plugins/server/io.github.flaxoos/ktor-server-kafka-jvm/2.0/application.conf

```diff
@@ -0,0 +1,30 @@
+ktor {
```

Comment:
Looks like you'll need some lines in the manifest file:

```
installation:
  default: install.kt
  application_conf: application.conf
  application_yaml: application.yaml
```

Reply:
Addressed and added another plugin